packages/workspace/src/generators/remove/lib/update-tsconfig.spec.ts

```diff
@@ -212,4 +213,42 @@ describe('updateTsconfig', () => {
       '@proj/nested/whatever-name': ['libs/my-lib/nested-lib/src/index.ts'],
     });
   });
+
+  it('should work with tsSolution setup', async () => {
+    jest.spyOn(tsSolution, 'isUsingTsSolutionSetup').mockReturnValue(true);
+
+    await libraryGenerator(tree, {
+      directory: 'my-lib',
+    });
+
+    const tsconfigContent = {
+      ...readJson(tree, 'tsconfig.base.json'),
```

Comment:
I don't think we need the entire content of `tsconfig.base.json`.  The base file needs to exist and only set `compilerOptions`, and the `tsconfig.json` file should have empty `include` and `files` to be considered TS solution